a11y-speak.js

```diff
@@ -76,6 +76,8 @@ var A11ySpeak = function( message, ariaLive ) {
 	// Clear previous messages to allow repeated strings being read out.
 	clear();
 
+	message = message.replace( /<[^<>]+>/g, "" );
```

Comment:
Is there a situation where we would use multiple tags right after each other where it would be possible 2 words are put together? In that case replacing with nothing could be causing an issue.